certbot/cli.py

```diff
@@ -1240,6 +1240,8 @@ def _plugins_parsing(helpful, plugins):
                       'using Google Cloud DNS).'))
     helpful.add(["plugins", "certonly"], "--dns-nsone", action="store_true",
                 help='Obtain certs using a DNS TXT record (if you are using NS1 for DNS).')
+    helpful.add(["plugins", "certonly"], "--dns-route53", action="store_true",
+                help='Obtain certs using a DNS TXT record (if you are using Route53 for DNS).')
```

Comment:
nit: There was a push in #4560 to consistently use "certificate" instead of "cert". I made a PR to fix this for NS1, but we should fix this for route53 here.